README.md

```diff
@@ -1,2 +1,48 @@
 # xapi-profiles
 A set of documents addressing the structure of and supporting services for xAPI Profiles.
+
+This Github repository contains the xAPI Profiles Specification. [xAPI](https://github.com/adlnet/xAPI-Spec) is a learning
+technologies interoperability specification that describes communication
+about learner activity and experiences between technologies. The specification stands as a 
+companion to xAPI, and is divided into three documents:
+
+*   Part One:   [About xAPI Profiles](./xapi-profiles-about.md)
+*   Part Two:   [xAPI Profiles Structure](./xapi-profiles-structure.md)
+*   Part Three: [xAPI Profiles Communication](./xapi-profiles-communication.md)
+
+## Specification versions
+
+The current version of the specification is
+[1.0](https://github.com/DataInteroperability/xapi-profiles).
+
+The next version of the specification has not been planned. 
+
+## What to do if the spec is unclear
+
+If when implementing the specification you find something is unclear or
+unhelpful, you can help to improve the xAPI Profiles specification by
+[raising an issue here](https://github.com/DataInteroperability/xapi-profiles/issues). When
+raising an issue, please give as much detail as you can in regards to:
+
+* Which part/parts of the specification you are reading.
+* Your understanding of what these parts mean.
+* The product and feature you are implementing xAPI in; what's the use case you
+  are trying to achieve?
+* How you would like the specification to be improved. Suggest some specific new
+  wording if you like!
+
+You'll need to [sign up for a GitHub account](https://github.com/signup/free) if
+you do not already have one in order to raise and comment on issues.
+
+You can discuss any issues before or after raising them on the
+[xAPI Profiles Google Group](https://groups.google.com/a/adlnet.gov/forum/#!forum/xapi-spec)
+and at ADL's
+[weekly specification calls](https://attendee.gotowebinar.com/register/5526804432322315009).
+
+## How you can contribute
+
+You'll need to [sign up for a GitHub account](https://github.com/signup/free) if
+you do not already have one in order to contribute to the specification.
+
+ADL's xAPI Spec Working Group meets
+the first, second, and third Wednesdays of each month on GoToWebinar. You can register at [https://attendee.gotowebinar.com/register/5526804432322315009](https://attendee.gotowebinar.com/register/5526804432322315009)
```

Comment:
Is there another page we can link to since this one becomes old every 50 weeks? 

(This also applies to the xAPI spec readme!)